packages/common/src/ui/components/portals/AppBarContent/AppBarContent.tsx

```diff
@@ -17,14 +17,22 @@ const Container = styled('div')({
 // mounted at any one time as mounting this in multiple locations would cause
 // some pretty weird behaviour
 export const AppBarContent: FC = () => {
-  const { appBarContentRef } = useHostContext();
+  const { setAppBarContentRef } = useHostContext();
+  const ref = useRef(null);
+
+  useEffect(() => {
+    setAppBarContentRef(ref);
+    () => setAppBarContentRef(null);
```

Comment:
how is this working without returning the function? the test passes, so something is happening. 😕